i18n/portuguese.json

```diff
@@ -451,10 +451,10 @@
     "cmd-giverole-remove" : "Eu removi a função {} das funções de {}.",
     "cmd-setrole-reset" : "A partir de agora não há mais funções que podem ser obtidas pelo comando {}!",
     "cmd-setrole-already" : "Essa função já foi obtida.",
-    "cmd-setrole-added" : "Nova função ue pode ser obtida adicionada.",
-    "cmd-setrole-removed" : "Esse papelo foi removido do papel This role has been remove from the role that can be obtained.",
+    "cmd-setrole-added" : "Novo função que pode ser obtida adicionada.",
+    "cmd-setrole-removed" : "Esse papél foi removido dos papéis disponíveis.",
     "cmd-setrole-already-d" : "Esta função não pode ser obtida",
-    "cmd-setrole-list" : "**Lista de papéis disponíveis:**\n",
+    "cmd-setrole-list" : "**Obtainable roles list:**\n",
```

Comment:
wut

Reply:
Oops... It should be "Lista de papéis disponíveis"